index.js

```diff
@@ -191,12 +213,16 @@ FocusGroup.prototype._findIndexOfNode = function(searchNode) {
 };
 
 FocusGroup.prototype._getActiveElementIndex = function() {
-  return this._findIndexOfNode(document.activeElement);
+  return this._rootNode !== document
+    ? this._activeIndex
+    : this._findIndexOfNode(document.activeElement);
 };
 
 FocusGroup.prototype.focusNodeAtIndex = function(index) {
   var member = this._members[index];
   if (member) focusNode(member.node);
```

Comment:
Is the only reason that the nodes in your example do *not* receive focus because they are not focusable elements? This seems like an important detail that should be specified via options, rather than the side-effect of element choice.